keyboards/manta60/keymaps/default/rules.mk

```diff
@@ -0,0 +1,13 @@
+# Build Options
+#   change to "no" to disable the options, or define them in the Makefile in
+#   the appropriate keymap folder that will get included automatically
+#
+# If your custom manta60 pcb, you can rewrite to yes.
+
+RGBLIGHT_ENABLE = yes  # LED underglow (Enable WS2812 RGB underlight.)
```

Comment:
This is already defined in `keyboards/manta60/rules.mk`, you don't need it here.

```suggestion
```